bin/internal/shared.sh

```diff
@@ -66,8 +66,9 @@ function update_flutter() {
   unset GIT_WORK_TREE
 
   # Invalidate the flutter cache.
+  local compilekey="$version:"
```

Comment:
`.stamp` does not contain `:`. Will the stamp information also change? Or is below commands no longer needed?

Reply:
No. The format has already changed as you can see in the above linked commit. We just didn't notice because there was no visible difference (due to `> /dev/null`).
```sh
$ cat flutter/bin/cache/flutter_tools.stamp
e1e47221e86272429674bec4f1bd36acc4fc7b77:
```